modules/sdk-core/src/bitgo/environments.ts

```diff
@@ -141,7 +141,7 @@ const testnetBase: EnvironmentTemplate = {
   serverXpub: hardcodedPublicKeys.serverXpub.test,
   hsmXpub: hardcodedPublicKeys.hsmXpub.test,
   btcExplorerBaseUrl: 'https://blockstream.info/testnet/api',
-  etherscanBaseUrl: 'https://api-goerli.etherscan.io',
+  etherscanBaseUrl: 'https://api-holesky.etherscan.io',
```

Comment:
Should we replace or add another url for holesky till the time we depreciate gteth because this is mostly used for recovery purpose. Means once this gets replaced WRW recoveries for gteth will stop working

Reply:
I think there should not be any usecase of actual funds recovering, it mostly is required to verify whether wrw is working for eth so it should be fine to override. Confirmed it with Daniel